### ISSUE TITLE
WSI encoding in BUFR

### ISSUE THREAD

**Issue by efucile** [wmo-im/WSI#1]
How WSI has to be encoded in BUFR is going to be addressed by TT-TDCF in this issue [BUFR4 issue 50](https://github.com/wmo-im/BUFR4/issues/50)

**Comment by efucile** [comment]
The main requirement for this task is that WSI is the new official unique identifier for a station and we need to have provisions on how to encode a BUFR message from a new station having only WSI.

**Comment by remygiraud** [comment]
New stations will only have country-code based WSI. They won't have TSI. So, what part is missing here? BUFR encoding? Other?

**Comment by golfvert** [comment]
Do we agree to put https://github.com/wmo-im/BUFR4/files/5175066/CircularLetter_2017-10-30-OBS-WIS-DRMM-DRC-WIGOS-ID_en.pdf into the Manual on Codes?

**Comment by jitsukoh** [comment]
I agree with the inclusion of part of the guidance presented in the circular letter (where we should place 3 01 150 sequence) but I think we need further consideration about having a "shall" statement in the note of all relevant sequences. I would invite TT-WSI team to consider what is the realistic migration from TSI to WSI for stations with TSI (and other traditional identifiers) before rushing into mandating WSI reporting in BUFR for everything.

**Comment by josusky** [comment]
@efucile Well, the WSI goes into 3 01 150 and the TSI part (3 01 001) is marked as missing. There is not much to discuss there, is it? :-)
Old stations, of course, fill their TSI into 301001, thus consequently report two station identifiers, to simplify transition.

**Comment by efucile** [comment]
We need to remember that there is only a circular letter on this, and we don't have provisions on the Manual on Codes.
TT-TDCF was not ready to provide any provision at this time, but it has to be done in the near future and it will be discussed again.
